test/math.jl

```diff
@@ -1020,6 +1020,11 @@ using .Main.Furlongs
         @test hypot(x,x/4) ≈ x * sqrt(BigFloat(17//16))
         @test hypot(x,x/4,x/4) ≈ x * sqrt(BigFloat(9//8))
     end
+    @testset "$T" for T in (Float16, Float32, Float64, BigFloat)
+        x = sqrt(nextfloat(zero(T)))*2
+        @test hypot(x,x/4) ≈ x * sqrt(17/BigFloat(16))
+        @test hypot(x,x/4,x/4) ≈ x * sqrt(9/BigFloat(8))
+    end
```

Comment:
See the updated (more stringent) version of this test in the comments.

Reply:
I left both versions.